package/boot/uboot-sunxi/Makefile

```diff
@@ -294,7 +295,6 @@ UBOOT_TARGETS := \
 	pangolin \
 	pine64_plus \
 	Sinovoip_BPI_M2_Plus \
```

Comment:
@CHKDSK88 BTW what about `Sinovoip_BPI_M2_Plus` ? Seems like it wont work these days.

```
ynezz@ntbk:/opt/devel/u-boot$ make Sinovoip_BPI_M2_Plus_config
***
*** Can't find default configuration "arch/../configs/Sinovoip_BPI_M2_Plus_defconfig"!
***
```

Reply:
@ynezz 

This is a reason: https://github.com/u-boot/u-boot/commit/268ae6548779ccd8ba38ce39d43f41be7e0bc133

It should be changed if we bump uboot-sunxi to >=2019.04.

Reply:
@CHKDSK88 Thanks, should be fixed via https://github.com/ynezz/openwrt/commit/fefa6f5a21a773c7fb434afe9f33012a4df7d815 as I plan to bump U-Boot to 2020.01 soon via https://github.com/ynezz/openwrt/commit/5cb06c0a62267bfb24f1a1c3d3fb6902d92db31b

Reply:
@ynezz 
Thanks for info. LGTM.
At this moment, I can't run test it because my banana was burned.